Sources_v3/StreamChatUI/MessageActionsPopup/ChatMessageActionItem.swift

```diff
@@ -26,7 +26,7 @@ public struct ChatMessageActionItem {
 public extension ChatMessageActionItem {
     static func inlineReply(action: @escaping () -> Void) -> Self {
         .init(
-            title: "Reply",
+            title: "message.actions.inline-reply".localized(),
```

Comment:
We need to generate these values from the strings file eventully. Ideally, we'd just do `Strings.message.actions.inlineReply` with autocompletion support :)

Reply:
Only I would stick with standard [L10n](https://en.wiktionary.org/wiki/L10n) instead of `Strings`

Reply:
So geeky...